### DISCUSSION TITLE
vc-drawing 绘制添加label问题

### DISCUSSION THREAD

**Discussion by xiaowunimte** [zouyaoji/vue-cesium#616] [General]
![1729477283718](https://github.com/user-attachments/assets/1a7334e2-93d0-4136-8103-dba7fb1160a9)
这个label属性，在绘制的时候可以在每个点上添加label吗

**Comment by zouyaoji** [comment]
应该是可以的哈，建议看看源码，绘制的话要把showLabel开启了，并且有text参数才会往渲染数组加标签。
https://github.com/zouyaoji/vue-cesium/blob/dev/packages/composables/use-drawing/use-drawing-point.ts#L556C17-L556C26
